pricenode/README.md

```diff
@@ -67,6 +67,21 @@ curl http://localhost:8080/getVersion
 curl http://localhost:8080/info
 ```
 
+### Monitoring
+
+If you run a main pricenode, you also are obliged to activate the monitoring feed by running
+
+```bash
+curl -s https://raw.githubusercontent.com/bisq-network/bisq/master/monitor/install_collectd_debian.sh | sudo bash
+```
+Follow the instruction given by the script and report your certificate to the seednode group!
```

Comment:
```suggestion
Follow the instruction given by the script and report your certificate to the pricenode group!
```

What does "group" mean here?

Reply:
```suggestion
Follow the instruction given by the script and report your certificate to the monitoring group via [github](https://github.com/orgs/bisq-network/teams/monitoring-operators) or keybase (https://keybase.io/team/bisq channel #monitoring)!
```

Reply:
"... report your certificate to the [`@bisq-network/monitoring`](https://github.com/orgs/bisq-network/teams/monitoring-operators) team or via the [Keybase](https://keybase.io/team/bisq) `#monitoring` channel."